lldb/tools/lldb-dap/README.md

```diff
@@ -204,12 +171,67 @@ The default hostname being used `localhost`.
 }
 ```
 
-## Custom debugger commands
+### Configuration Settings Reference
+
+For both launch and attach configurations, lldb-dap accepts the following `lldb-dap`
+specific key/value pairs:
+
+|parameter          |type|req |         |
```

Comment:
For `enableSyntheticChildDebugging`, this is on purpose. I am considering to rename it, because it not only useful for debugging the synthetic child providers, but also to debug memory corruptions

For `enableDisplayExtendedBacktrace` and `enableDisplayExtendedBacktrace `, I was considering to rename it and remove the `enable` prefix. The `enable` seems to be a filler word which adds no additional information. I didn't want to d those changes in this commit, though, since I wanted to keep this commit as a NFC

Reply:
After rebasing on `main`, I realized that `enableDisplayExtendedBacktrace` and `enableDisplayExtendedBacktrace` and `enableAutoVariableSummaries` are already documented on `main`

As such, I keep them unchanged in this commit now, and might do the rename separately